.gitignore

```diff
@@ -1,2 +1,3 @@
 gotags
 *.png
+.*.sw*
```

Comment:
Please remove this, I'd like to only keep repo specific ignores in this file. For editor related ignores you could use [core.excludesFile](https://git-scm.com/docs/git-config#git-config-coreexcludesFile) in your git config.